Add note about Deno.exit() API in worker contexts (#336)
denoland/manual: runtime/workers.md

hello world
```

> > Starting in v1.23 the `Deno.exit()` API no longer exits the process with the

Reviewer: ```suggestion
> Starting in v1.23 `Deno.exit()` no longer exits the process with the
```

> > provided exit code, but intead is an alias to `self.close()`, which causes the

Reviewer: ```suggestion
> provided exit code. Instead is an alias to `self.close()`, which causes only the
```

> > worker to shutdown, but keeps the program alive. This better aligns with the

Reviewer: ```suggestion
> worker to shutdown. This better aligns with the
```

> > Web platform, as there's no way in the browser for a worker to close the page

Reviewer: ```suggestion
> Web platform, as there is no way in the browser for a worker to close the page.
```

> > it was started on.

Reviewer: ```suggestion
```
